numpy/typing/tests/data/reveal/ctypeslib.pyi

```diff
@@ -20,7 +20,7 @@ AR_double: npt.NDArray[np.double]
 AR_longdouble: npt.NDArray[np.longdouble]
 AR_void: npt.NDArray[np.void]
 
-pointer: ctypes.pointer[Any]
+pointer: ctypes._Pointer[Any]
```

Comment:
Adapt to the `ctypes.pointer` changes in https://github.com/python/typeshed/pull/8446.